lib/phase-list.js

```diff
@@ -110,7 +112,7 @@ PhaseList.prototype.addAt = function(index, phase) {
 PhaseList.prototype.addAfter = function(after, phase) {
   var ix = this.getPhaseNames().indexOf(after);
   if (ix === -1) {
-    throw new Error('Unknown phase: "' + after + '"');
+    throw new Error(g.f('Unknown phase: %', after));
```

Comment:
@superkhau : this should be:
`throw new Error(g.f('Unknown phase: %s', after));`